src/main/java/org/jnode/fs/xfs/Superblock.java

```diff
@@ -62,11 +62,19 @@ public int getBlockSize() {
         return (int) getUInt32(0x4);
     }
 
+    public long getTotalBlocks() throws IOException {
+        return (int) getInt64(0x8);
+    }
+
+    public long getFreeBlocks() throws IOException {
+        return (int) getInt64(144);
+    }
     /**
      * Gets the UUID for the volume stored in the super block.
      *
      * @return the UUID.
      */
+
```

Comment:
Move this space to above the comment block above.

Reply:
code change applied